src/js/menu-cards.js

```diff
@@ -9,7 +9,15 @@ document.addEventListener('DOMContentLoaded', () => {
         card.addEventListener("click", () => {
             const title = card.querySelector("h4").textContent.trim();
 
-            fetch("http://localhost:3000/menuHighlights")
+            const baseUrl = process.env.NODE_ENV === 'development'
+                ? 'http://localhost:3000'
+                : '';
+
+            const url = process.env.NODE_ENV === 'development'
```

Comment:
Maak je `getApiUrl` functie herbruikbaar in een apart bestand `utils.js` bijvoorbeeld. Dan kan je hem hier gebruiken.